pages/services/edge-lb/1.0.0/api-reference/index.md

```diff
@@ -10,45 +10,8 @@ enterprise: false
 
 The Edge-LB API enables users to create and manage pools of load balancers.
 
-# Compatibility
+# API Versions
 
-The Edge-LB API was initially released alongside DC/OS 1.10.0 and requires DC/OS Enterprise 1.10.0 or newer.
+A new top level configuration field named `apiVersion` was introduced in Edge-LB v1.0.0. The two models are almost identical with one important difference: `pool.haproxy.backends.servers` (apiVersion `V1`) has been replaced with `pool.haproxy.backends.services` with a more intuitive way to select services / backends for HAProxy.
```

Comment:
Can we augment this description with something like:
"NOTE: Edge-LB 1.0 supports both the v1 and v2 API for backwards compatibility. Therefore clients that were written against Edge-LB versions prior to Edge-LB 1.0 should work with any modifications with Edge-LB 1.0"